lemmatized_text/parsers.py

```diff
@@ -18,13 +19,12 @@ def __init__(self, token_lemma_dict=None, lang=None):
     def handle_starttag(self, tag, attrs):
         if tag == "span":
             self.current_tag = "span"
-            self.current_attrs = dict(attrs)
+            self.current_attrs = json.loads(attrs["data-token"])
```

Comment:
Note that the HTML escaping will need to be reversed before parsing the JSON, otherwise I think this will return a `JSONDecodeError`. Since `html.escape()` was used above, you should be able to do an [html.unescape()](https://docs.python.org/3/library/html.html#module-html).